lang/pypy/files/getsysctlbyname.diff

```diff
@@ -0,0 +1,142 @@
+diff --git science/sumo/Portfile science/sumo/Portfile
+--- science/sumo/Portfile
++++ science/sumo/Portfile
```

Comment:
Why is this in pypy/files?

Reply:
A double mistake: 1) The patchfile isn't used, and should've been deleted and 2) I somehow had piped the wrong command into it.

Fixed.